README.md

```diff
@@ -81,15 +103,41 @@ def create_changeset(%User{} = user, attrs) do
 end
 ```
 
-### 3: Enjoy!
+### 4: Configuration
+
+Add in your configuration file.
+
+```elixir
+config :task_after, global_name: TaskAfter
+```
+
+### 4.1: S3 Bucket Configuration(required for S3 storage only)
+
+Add in your configuration file.
+
+```elixir
+config :ex_aws, :s3,
+  access_key_id: "key",
+  secret_access_key: "secret",
+  region: "us-east-1",
+  host: "localhost",
+  port: "9000",
+  scheme: "http://"
+
+config :my_project, :uploads,
+  bucket: "uploads",
+  region: "us-east-1"
+```
+
+### 5: Enjoy!
 
 Now, you can use your defined Uploader to handle your records with their files!
 
 The `use Uploadex` line in your Uploader module will import 3 groups of functions:
 
 #### Context
 
-  The highest level functions are context helpers (see [Context]([Resolver](https://hexdocs.pm/uploadex/Uploadex.Context.html#content)) for more documentation), which will allow you to easily create, update and delete your records with associated files:
+  The highest level functions are context helpers (see [Context](https://hexdocs.pm/uploadex/Uploadex.html#content) for more documentation), which will allow you to easily create, update and delete your records with associated files:
```

Comment:
Actually, this is a bit outdated. Everything goes through the Uploadex module now, the context helpers are here: https://hexdocs.pm/uploadex/3.0.0-rc.1/Uploadex.Context.html